__tests__/controller-test.js

```diff
@@ -0,0 +1,91 @@
+import * as controller from '../controller';
+
+jest.mock('../request');
+
+describe('Valid account returns properly', () => {
+  it('getAccountInfo returns an object with account info', () => {
+    expect.assertions(1);
+    return controller.getAccountInfo('2QW30682').then(accountInfo => expect(typeof accountInfo).toEqual('object'));
+  });
+
+  it('account object returns the correct user account', () => {
+    expect.assertions(1);
+    return controller.getAccountInfo('2QW30682').then(accountInfo => expect(accountInfo.account_number).toEqual('2QW30682'));
+  });
+
+  it('account object returns the correct user buying_power', () => {
+    expect.assertions(1);
+    return controller.getAccountInfo('2QW30682').then(accountInfo => expect(accountInfo.buying_power).toEqual('48642291.2050'));
+  });
+
+  it('account object returns the correct user option level', () => {
+    expect.assertions(1);
+    return controller.getAccountInfo('2QW30682').then(accountInfo => expect(accountInfo.option_level).toEqual(3));
+  });
```

Comment:
If these tests seem to be slowing things down, we could write a test with expect.assertions(2) or expect.assertions(3) (waiting for multiple callbacks to finish altogether). However it would be less clear which piece of the code is actually breaking unless you were careful about which tests to combine. (e.g. querying the same database, toEqual vs toThrow tests.)